app/src/main/java/com/tavarus/artabletop/Controllers/BoardController.kt

```diff
@@ -0,0 +1,27 @@
+package com.tavarus.artabletop.Controllers
+
+import android.content.Context
+import com.google.ar.sceneform.Node
+import com.tavarus.artabletop.Objects.Board
+import com.tavarus.artabletop.POJO.BoardListPOJO
+
+class BoardController(){
+    companion object {
+        var INSTANCE: BoardController = BoardController()
+    }
+
+    var currentBoard: Board? = null
+
+    fun initialize(list: BoardListPOJO, context: Context) {
+        currentBoard = Board(0.05f, list.boards[0], context)
```

Comment:
It isn't immediately clear what the first two arguments are for, but can you make the first one have a default? If there's a need to have it in the constructor, but you are typically constructing it with this constant, you should make that the default.

Reply:
It's tile size! Yeah it makes sense for it to have a default now. In the future, it could be changed based on user preference, but for now it's pretty static.